### ISSUE TITLE
Remove duplicate of client from server module

### ISSUE THREAD

**Issue by michaz** [graphhopper/graphhopper#1104]
com.graphhopper.http.GraphHopperWeb seems to be a duplicate of the "hand-coded" API client. Let's remove it. If it's there because of the integration tests: They can either use a test-scoped dependency to the actual client, or use a generic http client instead.

**Comment by michaz** [comment]
Prerequisite: hand-coded API client is Maven-deployed.

**Comment by boldtrn** [comment]
Ah, I think this would not work. The hc client depends on the GH core, therefore we get a circular dependency.

**Comment by michaz** [comment]
> test-scoped 
> circular dependency

Isn't that allowed?

**Comment by michaz** [comment]
If that's too brittle, no problem, integration test goes in its own module.
